src/pages/not-found/index.jsx

```diff
@@ -0,0 +1,48 @@
+import * as React from "react";
+import Box from "@mui/material/Box";
+import { useTranslation } from "react-i18next";
+
+export default function Insights() {
+  const { t } = useTranslation();
+  return (
+    <center>
+      <Box
+        component="img"
+        sx={{
+          width: { xs: "100%", md: "60%" },
+          height: "auto",
+        }}
+        alt="404 not found"
+        src={require("../../assets/404.png")}
+      />
+      <Box
```

Comment:
Boxes are a convenient way to structure your page with `div`. But here you are adding text. So in HTML it should be `<p>`.

You have a convenient way to do that in MUI [Typography](https://mui.com/material-ui/react-typography/#main-content) In which you play mainly with 2 props: `variant` for the styling and `component` for the web sementic.